_includes/why_d4a.md

```diff
@@ -0,0 +1,113 @@
+The [Docker for
+{{cloudprovider}}](https://docs.docker.com/docker-for-{{cloudprovider |
+downcase}}/) project was created and is being actively developed to ensure that
+Docker users can enjoy a fantastic out-of-the-box experience on
+{{cloudprovider}}.  It is now generally available and can now be used by
+everyone.
+
+As an informed user, you might be curious to know what this project has to offer
+you for running your development, staging, or production workloads.
+
+## Native to Docker
+
+Docker for {{cloudprovider}} was built with a passion for simplicity and great
+user experience. We believe strongly that additional complexity must be
+justified by substantial additional benefit.  Some solutions are very complex to
+operate or layer unneeded additional APIs on top of Docker, but Docker for
+{{cloudprovider}} provides a Docker-native solution.
+
+Instead of having to learn one or more layers on top of Docker, with Docker for
+{{cloudprovider}} you will interact with Docker (including native Docker
+orchestration) directly.  These setups free you to focus on the thing that
+matters most: running your workloads. This will help you and your team to
+deliver more value to the business faster, to speak one common "language", and
+to have fewer details to keep in your head at once.
+
+The skills that you and your team have already learned, and will continue to
+learn, using Docker on the desktop or elsewhere will automatically carry over to
+using Docker on {{cloudprovider}} and vice versa.  The added consistency across
+clouds also helps to ensure that a migration or multi-cloud strategy would be
+easier to accomplish in the future if desired.
+
+## Skip the Boilerplate and Maintenance Work
+
+Docker for {{cloudprovider}} bootstraps all of the recommended infrastructure to
+start using Docker on {{cloudprovider}} automatically.  You won't need to worry
+about rolling your own instances, security groups, load balancers, and so on
+when using Docker for {{cloudprovider}}.
+
+Likewise, the setup of the Docker swarm mode functionality (vital for container
```

Comment:
Likewise, setting up and using Docker swarm mode functionality for container orchestration is managed across the cluster's lifecycle when you use Docker for _____. Docker has already